editor/test/selectors.js

```diff
@@ -839,4 +890,57 @@ describe( 'selectors', () => {
 			expect( isSavingNewPost( state ) ).to.be.false();
 		} );
 	} );
+
+	describe( 'getSuggestedPostFormat', () => {
+		it( 'returns null if cannot be determined', () => {
```

Comment:
>Wording of these test names?

We just talked about this at https://github.com/WordPress/gutenberg/pull/946#issuecomment-305277136 😛  I'd argue this is one case where I wish we _weren't_ so consistent, because bending over backwards to prefix with "should" isn't always appropriate and at worst is unnecessary filler.